RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -0,0 +1,316 @@
+{
+	return canUse(sk, true);
+}
+
+boolean canUse(item it, boolean onlyOnce)
+{
+	if(onlyOnce && haveUsed(it))
+		return false;
+
+	if(item_amount(it) == 0)
+		return false;
+
+	if(!auto_is_valid(it))
+		return false;
+
+	return true;
+}
+
+boolean canUse(item it) // assume onlyOnce unless specified otherwise
+{
+	return canUse(it, true);
+}
+
+string useSkill(skill sk, boolean mark)
+{
+	if(mark)
+		markAsUsed(sk);
+
+	return "skill " + sk.name;
+}
+
+string useSkill(skill sk)
+{
+	return useSkill(sk, true);
+}
+
+string useItem(item it, boolean mark)
+{
+	if(mark)
+		markAsUsed(it);
+	return "item " + it;
+}
+
+string useItem(item it)
+{
+	return useItem(it, true);
+}
+
+string useItems(item it1, item it2, boolean mark)
+{
+	if(mark)
+	{
+		markAsUsed(it1);
+		markAsUsed(it2);
+	}
+	return "item " + it1 + ", " + it2;
+}
+
+string useItems(item it1, item it2)
+{
+	return useItems(it1, it2, true);
+}
+
+string getStallString(monster enemy)
+{
+	// if you call this without actually doing what it returns it'll make me sad :(
+	
+	if(canUse($skill[Summon Love Gnats]))
+	{
+		return useSkill($skill[Summon Love Gnats]);
+	}
+
+	if(canUse($skill[Beanscreen]))
+	{
+		return useSkill($skill[Beanscreen]);
+	}
+
+	if(canUse($skill[Broadside]))
+	{
+		return useSkill($skill[Broadside]);
+	}
+
+	if(canUse($skill[Mind Bullets]))
+	{
+		return useSkill($skill[Mind Bullets]);
+	}
+
+	if(canUse($skill[Snap Fingers]))
+	{
+		return useSkill($skill[Snap Fingers]);
+	}
+
+	if(canUse($skill[Soul Bubble]))
+	{
+		return useSkill($skill[Soul Bubble]);
+	}
+
+	if(canUse($skill[Blood Chains]))
+	{
+		return useSkill($skill[Blood Chains]);
+	}
+
+	if(canUse($skill[Hogtie]) && !haveUsed($skill[Beanscreen]) && (my_mp() >= (6 * mp_cost($skill[Hogtie]))) && hasLeg(enemy))
+	{
+		return useSkill($skill[Hogtie]);
+	}
+
+	return "";
+}
+
+boolean enemyCanBlocksSkills()
+{
+	//we want to know if enemy can sometimes block a skill. For such enemies skills should be used only if absolutely necessary
+	//for enemies that always block a skill a seperate function should be made... if we ever fight any in run.
+	
+	monster enemy = last_monster();
+	
+	if($monsters[
+	Bonerdagon,
+	Naughty Sorceress,
+	Naughty Sorceress (2)
+	] contains enemy)
+	{
+		return true;
+	}
+	
+	return false;
+}
+
+boolean registerCombat(string action)
+{
+	set_property("auto_combatHandler", get_property("auto_combatHandler") + "(" + to_lower_case(action) + ")");
+	return true;
+}
+
+boolean registerCombat(skill sk)
+{
+	return registerCombat(to_string(sk));
+}
+
+boolean registerCombat(item it)
+{
+	return registerCombat(to_string(it));
+}
+
+boolean containsCombat(string action)
+{
+	action = "(" + to_lower_case(action) + ")";
+	return contains_text(get_property("auto_combatHandler"), action);
+}
+
+boolean containsCombat(skill sk)
+{
+	return containsCombat(to_string(sk));
+}
+
+boolean containsCombat(item it)
+{
+	return containsCombat(to_string(it));
+}
```

Comment:
These functions are not only not used any more, they're also just plain wrong since I long ago switched the format to be `(sk4034)` instead of `(Curse of Weaksauce)` and `(it9104)` instead of `(Time-Spinner)` (those numbers being the corresponding skill or item `.to_int()`). Please remove them.

Reply:
Oh, I didn't notice the `registerCombat` functions right above them that are likewise no longer used and incorrect. Please remove those too.

Reply:
deleted those functions as requested